### ISSUE TITLE
Introduce the idea of "mixers" or small group chats to help bring back kotters

### ISSUE THREAD

**Issue by kofkne** [F3Nation-Community/weaselbot#12]
Original Post from slack (Griswold, F3xT Cross Timbers)
Background: The Greetbot app has a feature to automatically send messages to groups of three people once every two weeks in a rotating, mixing fashion, with idea being, "Hey, you three fellas go get coffee. Get to know each other better."
Issue: Greetbot didn't filter inactive slack users. Thus, the messages often fell on deaf ears and were ignored. An improvement would be the same functionality as above, but with the active filter.

Possibly integrate the kotters to have besties of the kotter get this mixer message

**Comment by jasheldo** [comment]
I think this is similar to #14. Could probably be tied in with that one.

**Comment by kofkne** [comment]
yes - it could be tied in with that one... this was more specific to a way to get pax together with random get togethers... but I like the achievements version of it a little better
